resview.py

```diff
@@ -633,6 +654,9 @@ def main():
     parser.add_argument('--off-screen',
                         action='store_true', dest='off_screen',
                         default=False, help=helps['off_screen'])
+    parser.add_argument('-x', '--xy-view',
```

Comment:
Maybe remove the single-letter option? It seems ambiguous.

Reply:
Yes, It can be confusing, but `--xy-view` is rather long and `-x` does not collide with anything else. So I would keep it. It is not the best, like `t` is not, but I do not have a better suggestion.

Reply:
OK (but see below).

Note that you need not to write the complete option name if it does not coincide with another option - so writing `--xy` would work.

Moreover, argparse allows specifying a number of variants rather freely, so the first/short option version could be `--xy` or even `-xy` explicitly. Not so long anymore :)

Reply:
So what about using `--plane_xy_view` and shorten to `-p`?

Reply:
`--plane-view`, `-p`? It depends if you would like to add other view options in future (xz, isometric, parallel projection, whatever pyvista can). The convert_mesh.py script has `-2`, `--2d` in a similar setting. Well, sorry for the bike-shedding, yours will be the last word :)

Reply:
Finally, `-2` and `--2d` is probably the best.

Reply:
.. or better: `--2d-view`

Reply:
OK!